Zotero/Scenes/Master/Settings/General/Views/GeneralSettingsView.swift

```diff
@@ -16,27 +16,49 @@ struct GeneralSettingsView: View {
             SettingsToggleRow(
                 title: L10n.Settings.General.showSubcollectionsTitle,
                 subtitle: nil,
-                value: self.viewModel.binding(keyPath: \.showSubcollectionItems, action: { .setShowSubcollectionItems($0) })
+                value: viewModel.binding(keyPath: \.showSubcollectionItems, action: { .setShowSubcollectionItems($0) })
             )
 
             SettingsToggleRow(
                 title: L10n.Settings.General.showCollectionItemCounts,
                 subtitle: nil,
-                value: self.viewModel.binding(keyPath: \.showCollectionItemCounts, action: { .setShowCollectionItemCounts($0) })
+                value: viewModel.binding(keyPath: \.showCollectionItemCounts, action: { .setShowCollectionItemCounts($0) })
             )
 
             SettingsToggleRow(
                 title: L10n.Settings.General.openLinksInExternalBrowser,
                 subtitle: nil,
-                value: self.viewModel.binding(keyPath: \.openLinksInExternalBrowser, action: { .setOpenLinksInExternalBrowser($0) })
+                value: viewModel.binding(keyPath: \.openLinksInExternalBrowser, action: { .setOpenLinksInExternalBrowser($0) })
             )
+
+            Picker(L10n.Settings.General.autoemptyTitle, selection: viewModel.binding(get: \.autoEmptyTrashThreshold, action: { .setAutoEmptyTrashThreshold($0) })) {
+                Text(L10n.Settings.General.afterOneDay).tag(1)
+                Text(L10n.Settings.General.afterXDays(7)).tag(7)
+                Text(L10n.Settings.General.afterXDays(15)).tag(15)
+                Text(L10n.Settings.General.afterXDays(30)).tag(30)
+                Text(L10n.Settings.General.never).tag(0)
+            }
         }
         .navigationBarTitle(L10n.Settings.General.title)
     }
+
+    private func thresholdString(for threshold: Int) -> String {
```

Comment:
This seems not used anywhere.

Reply:
Removed 5ae4b8c5aff31c30fcbbffc5e535f1eb194f18f7